docs/configuration.asciidoc

```diff
@@ -120,6 +120,18 @@ A boolean specifying if the agent should collect performance metrics for the app
 
 Note that both `active` and `instrument` needs to be `true` for instrumentation to be running.
 
+[[remote-config]]
+==== `remoteConfig`
+
+* *Type:* Boolean
+* *Default:* `false`
+* *Env:* `ELASTIC_APM_REMOTE_CONFIG`
+
+Activate APM Agent Configuration via Kibana.
+If set to `true`, the client will poll the APM Server regularly for new agent configuration.
+
+NOTE: This feature requires APM Server v7.3 or later.
```

Comment:
@bmorelli25 we probably want to align on this wording across all agents. And we probably want to also add that this feature is disabled by default in the APM Server (as far as I know). So a user needs v7.3 + they need to enable it in `apm-server.yml` before it works. I don't know if it's going to be enabled by default in Elastic Cloud -@roncohen do you know?

Reply:
I updated the line to include information about the APM Server config option (see updated version in PR)

Reply:
Forgot to mention: According to @simitt, the feature will be off by default in the APM Server for 7.3